srcpkgs/hstr/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'hstr'
 pkgname=hstr
-version=2.2
+version=2.3
 revision=1
 build_style=gnu-configure
 hostmakedepends="automake pkg-config"
 makedepends="readline-devel"
 short_desc="Easily view, navigate, search and manage your command history"
-maintainer="Diogo Leal <diogo@diogoleal.com>"
+maintainer="Neel Chotai <neel@chot.ai>"
 license="Apache-2.0"
 homepage="https://github.com/dvorka/hstr"
 changelog="https://raw.githubusercontent.com/dvorka/hstr/master/Changelog"
 distfiles="https://github.com/dvorka/hstr/archive/${version}.tar.gz"
-checksum=886531ca9d8d1a5a93a3db96e9fc64154a03f6de428c74c722b41057bceca266
+checksum=c7e7408671757b3f4be9c5a59b4e2d56e7a7b601ace2a94eb6b2b61f20ee890b
 
 pre_configure() {
 	grep -rli '#include <ncursesw/curses.h>' src | xargs -i@ sed -i 's|ncursesw/curses.h|curses.h|g' @
```

Comment:
Maybe replace this with the files from the PKGBUILD? https://github.com/dvorka/hstr/blob/60fa27920623dc74abeee5212fb8ca58c31e2b72/PKGBUILD

And use vsed as well.

Reply:
Done, looks like there were some instances of `<ncursesw/curses.h>` in tests that don't need to be replaced.